tests/python/pants_test/backend/python/tasks/test_python_distribution_integration.py

```diff
@@ -53,9 +56,25 @@ def test_pants_run(self):
       # Check that text was properly printed to stdout.
       self._assert_native_greeting(pants_run.stdout_data)
 
+  @staticmethod
+  def rewrite_hello_c_source(orig_content):
+    return re.sub('"Hello from C!"', '"Hello from C?"', orig_content)
```

Comment:
Since the usage of this function is fairly contained in the particular test below, I'd recommend moving it into the test itself. `@staticmethod` or not isn't important.

Reply:
Done, see the [new version of the test](https://github.com/cosmicexplorer/pants/blob/83429ecd725ef49f195a5e5bf6f9ee195f340049/tests/python/pants_test/backend/python/tasks/test_python_distribution_integration.py#L60).